app/controllers/users/sessions_controller.rb

```diff
@@ -31,10 +31,9 @@ def active
     end
 
     def timeout
-      if sign_out
-        analytics.track_event(Analytics::SESSION_TIMED_OUT)
-        flash[:timeout] = t('session_timedout')
-      end
+      analytics.track_event(Analytics::SESSION_TIMED_OUT)
+      sign_out
```

Comment:
I think the behavior is that if the user is already signed out, visiting `/timeout` does not show the flash message?